apps/lobe-chat/1.36.32/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.36.32
+    image: lobehub/lobe-chat:v1.36.33
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The code changes involve updating to v1.36.33 of the `lobehub/lobe-chat` Docker image from version v1.36.32. This update might include new features, bug fixes, or improvements. However, without additional context about what specific functionality is implemented in these versions, I cannot make any further analysis on whether there are potential issues that may arise due to this change.

I would recommend reviewing any release notes or documentation for v1.36.33, and testing the updated image if applicable, to ensure it meets your requirements or identifies any unexpected behaviors before deploying it widely.